### DISCUSSION TITLE
Supported OS versions overview

### DISCUSSION THREAD

**Discussion by hbugdoll** [appcelerator/titanium_mobile#13555] [General]
We miss a table in the guide which lists the supported iOS/iPadOS and Android major versions of each SDK release
(e.g. _fully supported_, _not fully supported_, _not supported_ or _full support planned_ for upcoming releases).

Often you have to trawl through many release notes and sometimes the result is vague.

**Comment by m1ga** [comment]
Android: https://titaniumsdk.com/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Titanium_Compatibility_Matrix/#supported-versions
XCode: https://titaniumsdk.com/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Titanium_Compatibility_Matrix/#supported-versions-of-xcode
iOS: https://titaniumsdk.com/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Titanium_Compatibility_Matrix/#supported-versions-of-ios-sdk-target-ios-platform

**Reply by m1ga** [reply]
and for Android I always look in the package.json: https://github.com/tidev/titanium_mobile/blob/master/android/package.json#L19-L27 as its a bit quicker to see :smile:

**Reply by m1ga** [reply]
Feel free to contribute if you are missing something: https://github.com/tidev/titanium-docs/tree/main/docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Titanium_Compatibility_Matrix 
would love to see some more contributions to make the docs better!

**Comment by hbugdoll** [comment]
@m1ga shame on me!
When I see the compatibility matrix I remember it.... too less sleep.

**Comment by hbugdoll** [comment]
Android 13 was first supported in Ti 12.0.0 (https://github.com/tidev/titanium-sdk/pull/13474).
Ti 12.1.0 comes with "Enhanced support for Android 13" (https://github.com/tidev/titanium-sdk/pull/13750, https://github.com/tidev/titanium-sdk/pull/13794, ...).

@m1ga Can one now say Ti 12.1.0 has "Full support for Android 13" in terms of the Xcode compatibility matrix https://titaniumsdk.com/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Titanium_Compatibility_Matrix/#supported-versions-of-xcode or are there features missing?

**Reply by m1ga** [reply]
"Full support" would be a very bold statement :smile: Don't think we have all the features of Android in the SDK. But we can add "Android 13 support" to it. Currently we don't say anything about that for Android besides that we target Android 33 by default now. I'll check where the best place for that should be
